lib/main.dart

```diff
@@ -1,11 +1,13 @@
 import 'package:firebase_core/firebase_core.dart';
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
 import 'package:flutter/material.dart';
 import 'package:movies_and_stars/di/app_injector.dart';
 import 'package:presentation/star_movie_app.dart';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   await Firebase.initializeApp();
+  FlutterError.onError = FirebaseCrashlytics.instance.recordFlutterFatalError;
```

Comment:
where do u decide to log crash events if mode = release?

Reply:
resolved